app/javascript/components/reusable/withUserForm.js

```diff
@@ -58,15 +58,20 @@ const withUserForm = (WrappedComponent, schema, wrappedProps) => {
       this.resetForm = this.resetForm.bind(this);
       this.changeCountryHandler = this.changeCountryHandler.bind(this);
       this.changeStateHandler = this.changeStateHandler.bind(this);
+<<<<<<< HEAD
```

Comment:
Can you remove this merge issue we have going on?

Reply:
Sure, will update when done.